testdata/install.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
The file could have the execution flag set to make it easier to execute by itself (i.e., `./install.sh` rather than `sh install.sh` or similar). Not a big deal though.